Flutter User Facing API.dart

```diff
@@ -0,0 +1,71 @@
+class BleManager {
+  Future<void> createClient(String restoreStateIdentifier) async
+  Future<void> destroyClient() async
+
+  Future<void> enableRadio()
+  Future<void> disableRadio()
+  
+  Future<void> cancelTransaction(String transactionId) async
+
+  Future<void> setLogLevel(LogLevel logLevel)
+  Future<LogLevel> logLevel()
+
+  Future<BluetoothState> state()
```

Comment:
Should this be a stream?

Reply:
This is a one time getter, as opposed to `onStateChange()`. Perhaps the other function should be renamed to `observeState()` and emit the current state immediately on creation?

Reply:
I would add an option to the user if he wants to emit initial value in the stream.